calc/vxlan_resolver.go

```diff
@@ -351,15 +358,15 @@ func (c *VXLANResolver) routeReady(r vxlanRoute) bool {
 	logCxt := logrus.WithField("route", r)
 	gw := c.determineGatewayForRoute(r)
 	if gw == "" {
-		logCxt.Info("No gateway yet for VXLAN route, skip")
+		logCxt.Debug("No gateway yet for VXLAN route, skip")
```

Comment:
I made these Info before but then realised they could be too spammy so backing them back down to debug.